dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeConfig.java

```diff
@@ -616,4 +645,82 @@ private static void validateKeyFilePath(String path, String type) throws AcmeCaE
 			}
 		}
 	}
+	
+	/**
+	 * 
+	 * @return the certCheckerScheduler
+	 */
+	@Trivial
+	public Long getCertCheckerScheduler() {
+		return certCheckerScheduler;
+	}
+
+	/**
+	 * Sets the certCheckerScheduler. If set to 0 or less, the certificate
+	 * checker is considered disabled. If set below the min renew time, reset to the
+	 * min renew time.
+	 * @param certCheckerScheduler
+	 */
+	public void setCertCheckerScheduler(Long certCheckerScheduler) {		
+		if (certCheckerScheduler != null) {
+			if (certCheckerScheduler <= 0) {
+				if (tc.isDebugEnabled()) {
+					Tr.debug(tc, "certChecker is disabled");
+				}
+				this.certCheckerScheduler = 0L;
+			} else if (certCheckerScheduler < AcmeConstants.RENEW_CERT_MIN) {
+				/*
+				 * Too low of a timeout, reset to the min renew allowed
+				 */
+				this.certCheckerScheduler = AcmeConstants.RENEW_CERT_MIN;
+				if (tc.isDebugEnabled()) {
+					Tr.debug(tc, "certCheckerScheduler is reset to the minimum: "+ this.certCheckerScheduler);
+				}
+			} else { 
+				this.certCheckerScheduler = certCheckerScheduler;
+			}
+		}
+	}
+
+	
+	/**
+	 * Get the certCheckerErrorScheduler
+	 * @return certCheckerErrorScheduler
+	 */
+	@Trivial
+	public Long getCertCheckerErrorScheduler() {
+		return certCheckerErrorScheduler;
+	}
+
+	/**
+	 * Set the certCheckerErrorScheduler
+	 * If it is set below the min renewal time, reset to the min renewal time
+	 *
+	 * @param certCheckerErrorScheduler
+	 */
+	public void setCertCheckerErrorScheduler(Long certCheckerErrorScheduler) {
+		if (certCheckerErrorScheduler != null) {
+			if (certCheckerErrorScheduler < AcmeConstants.RENEW_CERT_MIN) {
```

Comment:
I think you can define a minimum in the metatype. Not sure if that would work better. If you don't do it there, should you have a warning message here instead of debug?

Reply:
So you can, you learn something new every day